browser_history/cli.py

```diff
@@ -28,11 +29,20 @@ def make_parser():
     parser_ = argparse.ArgumentParser(
         description="""
                     A tool to retrieve history from
-                    (almost) any browser on (almost) any platform""",
+                    (almost) any browser on (almost) any platform
+
+██████╗ ██████╗  ██████╗ ██╗    ██╗███████╗███████╗██████╗       ██╗  ██╗██╗███████╗████████╗ ██████╗ ██████╗ ██╗   ██╗
+██╔══██╗██╔══██╗██╔═══██╗██║    ██║██╔════╝██╔════╝██╔══██╗      ██║  ██║██║██╔════╝╚══██╔══╝██╔═══██╗██╔══██╗╚██╗ ██╔╝
+██████╔╝██████╔╝██║   ██║██║ █╗ ██║███████╗█████╗  ██████╔╝█████╗███████║██║███████╗   ██║   ██║   ██║██████╔╝ ╚████╔╝
+██╔══██╗██╔══██╗██║   ██║██║███╗██║╚════██║██╔══╝  ██╔══██╗╚════╝██╔══██║██║╚════██║   ██║   ██║   ██║██╔══██╗  ╚██╔╝
+██████╔╝██║  ██║╚██████╔╝╚███╔███╔╝███████║███████╗██║  ██║      ██║  ██║██║███████║   ██║   ╚██████╔╝██║  ██║   ██║
+╚═════╝ ╚═╝  ╚═╝ ╚═════╝  ╚══╝╚══╝ ╚══════╝╚══════╝╚═╝  ╚═╝      ╚═╝  ╚═╝╚═╝╚══════╝   ╚═╝    ╚═════╝ ╚═╝  ╚═╝   ╚═╝
+                    """,  # noqa: E501
         epilog="""
                 Checkout the GitHub repo
                 https://github.com/pesos/browser-history
                 if you have any issues or want to help contribute""",
+        formatter_class=RawTextHelpFormatter,
```

Comment:
A small issue. Because of this, the newlines were also removed from the rest of the help section. So some of them look weird:
![image](https://user-images.githubusercontent.com/34161949/134011530-61ed230f-feae-41c1-8297-6590c2df2b09.png)

According to one of the comments [here](https://stackoverflow.com/a/3853776), using `RawDescriptionHelpFormatter` instead of `RawTextHelpFormatter` would fix this.

Reply:
ok